cmd/admin-handlers.go

```diff
@@ -1581,7 +1581,9 @@ func (a adminAPIHandlers) TraceHandler(w http.ResponseWriter, r *http.Request) {
 
 	// Trace Publisher and peer-trace-client uses nonblocking send and hence does not wait for slow receivers.
 	// Use buffered channel to take care of burst sends or slow w.Write()
-	traceCh := make(chan madmin.TraceInfo, 4000)
+
+	// Keep 100k buffered channel, should be sufficient to ensure we do not lose any events.
+	traceCh := make(chan madmin.TraceInfo, 100000)
```

Comment:
@harshavardhana If 4K overflows, 100K will overflow, though you may have a bit more time before it does (and eating more memory in the meantime).

We just have to accept that this can happen and that traces will be leaky under high load.

Reply:
Correct but 100k is sufficient enough.